src/helpers.ts

```diff
@@ -118,15 +120,56 @@ export function getOrCreateMarketDailySnapshot(
   blockTimestamp: i32,
 ): MarketDailySnapshot {
   let snapshotID = marketID.concat('-').concat(getEpochDays(blockTimestamp).toString())
+  log.info('[getorCreateMarketDailySnapshot] snapshotID: {}', [snapshotID])
   let snapshot = MarketDailySnapshot.load(snapshotID)
   if (!snapshot) {
+    log.warning('[getorCreateMarketDailySnapshot] snapshot not found, creating new snapshot {}', [snapshotID])
     snapshot = new MarketDailySnapshot(snapshotID)
     snapshot.market = marketID
     snapshot.totalBorrows = zeroBD
     snapshot.totalSupplies = zeroBD
     snapshot.totalBorrowsUSD = zeroBD
     snapshot.totalSuppliesUSD = zeroBD
     snapshot.save()
+
+    // Load the market entity
+    let market = Market.load(marketID)
+
+    // Iterate over the accountCToken IDs and create a snapshot for each one
+    if (market) {
+      for (let i = 0; i < market.accountCTokens.length; i++) {
```

Comment:
This type of iteration across an array doesn't work in GraphQL so I'm not sure how to do this.